python/activity_stream/activity_stream.py

```diff
@@ -145,6 +146,14 @@ def destroy(self):
     # properties
 
     @property
+    def note_threads(self):
+        """
+        The currently loaded note threads, keyed by Note entity id and
+        containing a list of Shotgun entity dictionaries.
```

Comment:
What are these note threads? Are they all the note threads that are expanded within the activity stream? Please extend the comment to give more context. Maybe also update the  class level doc string.